app/src/main/java/org/oppia/android/app/devoptions/markchapterscompleted/MarkChaptersCompletedFragmentPresenter.kt

```diff
@@ -72,12 +72,19 @@ class MarkChaptersCompletedFragmentPresenter @Inject constructor(
           if (viewModel is ChapterSummaryViewModel) {
             if (!viewModel.checkIfChapterIsCompleted())
               chapterSelected(
-                viewModel.chapterIndex,
-                viewModel.nextStoryIndex,
+                viewModel.chapterIndex, viewModel.nextStoryIndex,
```

Comment:
Undo this change.
Each argument should be in its own line here.

Reply:
This appears to be resolved, or at least is no longer an issue in the latest changes.